assets/js/modules/analytics-4/datastore/properties.js

```diff
@@ -0,0 +1,159 @@
+/**
+ * `modules/analytics-4` data store: properties.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import invariant from 'invariant';
+
+/**
+ * Internal dependencies
+ */
+import API from 'googlesitekit-api';
+import Data from 'googlesitekit-data';
+import { STORE_NAME } from './constants';
+import { createFetchStore } from '../../../googlesitekit/data/create-fetch-store';
+
+const fetchGetPropertiesStore = createFetchStore( {
+	baseName: 'getProperties',
+	controlCallback( { accountID } ) {
+		return API.get( 'modules', 'analytics-4', 'properties', { accountID }, {
+			useCache: true,
+		} );
+	},
+	reducerCallback( state, properties, { accountID } ) {
+		return {
+			...state,
+			properties: {
+				...state.properties,
+				[ accountID ]: properties,
+			},
+		};
+	},
+	argsToParams( accountID ) {
+		return { accountID };
+	},
+	validateParams( { accountID } = {} ) {
+		invariant( accountID, 'GA4 accountID is required.' );
```

Comment:
GA4 doesn't have different account IDs, the separation between the versions starts at the property-level.
```suggestion
		invariant( accountID, 'accountID is required.' );
```